content/en/docs/home/_index.md

```diff
@@ -56,9 +56,9 @@ cards:
   description: Anyone can contribute, whether you're new to the project or you've been around a long time.
   button: Contribute to the docs
   button_path: /docs/contribute
-- name: release-notes
-  title: K8s Release Notes
-  description: If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.
+- name: Download
+  title: Downloads
+  description: If you are installing Kubernetes or upgrading to the newest version, refer to the Downloads section.
```

Comment:
```suggestion
  description: Install Kubernetes or upgrade to the newest version.
```
The button is the call to action. "Refer to the Downloads section" doesn't do anything unless we link that text to the downloads section, but that makes the button redundant. The Training section has similar language, keeping the button as the call to action

Reply:
Thank you for suggestion!

Reply:
@shannonxtreme the changes have been applied now!